pkgs/development/compilers/llvm/common/clang-tools/wrapper

```diff
@@ -2,26 +2,43 @@
 
 buildcpath() {
   local path after
+
   while (( $# )); do
     case $1 in
         -isystem)
             shift
             path=$path${path:+':'}$1
             ;;
+
         -idirafter)
             shift
             after=$after${after:+':'}$1
             ;;
     esac
+
     shift
   done
+
```

Comment:
[nit] these whitespace changes should be a commit on their own, because they're not part of the change being made here.